packages/mobile/src/home/CashInBottomSheet.tsx

```diff
@@ -0,0 +1,104 @@
+import Button, { BtnSizes } from '@celo/react-components/components/Button'
+import Touchable from '@celo/react-components/components/Touchable'
+import Times from '@celo/react-components/icons/Times'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+// import styles from '@celo/react-components/styles/styles'
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import Modal from 'react-native-modal'
+import { FiatExchangeEvents, HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { Namespaces } from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+
+function CashInBottomSheet() {
+  const { t } = useTranslation(Namespaces.walletFlow5)
+  const [isModalVisible, setModalVisible] = useState(true)
+
+  useEffect(() => {
+    ValoraAnalytics.track(HomeEvents.impression_cash_in_bottom_sheet)
+  })
```

Comment:
`useEffect` takes a dependency array as a second parameter, which tells the component when to run the effect. When no dependency array is included, then the function is called on every re-render, which happens quite frequently for a typical React component.

In this instance, we only want to track analytics once when this component is initially rendered. In order to accomplish this, we should pass in an empty array `[]` as the dependency array. This will ensure we are only logging the analytics event once per session